src/App.tsx

```diff
@@ -1,56 +1,105 @@
+    setQuery(event.target.value);
+    applyQuery(event.target.value);
+  };
+
+  const handleItemClick = (person: Person) => {
+    setSelectedPerson(person);
+    setQuery(person.name);
+  };
 
   return (
     <main className="section">
-      <h1 className="title">
-        {`${name} (${born} = ${died})`}
-      </h1>
+      {selectedPerson
+        ? (
+          <h1 className="title">
+            {`${selectedPerson.name} (${selectedPerson.born} = ${selectedPerson.died})`}
+          </h1>
+        )
+        : (
+          <h1 className="title">No selected person</h1>
+        )}
+
+      <br />
 
       <div className="dropdown is-active">
         <div className="dropdown-trigger">
           <input
             type="text"
             placeholder="Enter a part of the name"
             className="input"
+            value={query}
+            onChange={handleQueryChange}
+            onFocus={() => setShowDropDownMenu(true)}
+            onBlur={() => setShowDropDownMenu(false)}
           />
         </div>
 
-        <div className="dropdown-menu" role="menu">
-          <div className="dropdown-content">
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Bernard Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Antone Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Haverbeke</p>
+        {showDropDownMenu && (
+          <div className="dropdown-menu" role="menu">
+            <div className="dropdown-content">
+              {filteredPeople.length
+                ? filteredPeople.map(person => (
+                  <div className="dropdown-item" key={person.slug}>
+                    <a
+                      href={`#${person.slug}`}
+                      onMouseDown={() => handleItemClick(person)}
+                      className={classNames(
+                        { 'has-text-link': person.sex === 'm' },
+                        { 'has-text-danger': person.sex === 'f' },
+                      )}
+                    >
+                      {person.name}
+                    </a>
+                  </div>
+                ))
+                : (
+                  <div className="dropdown-item">
+                    <p>
+                      No matching suggestions
+                    </p>
+                  </div>
+                )}
             </div>
+          </div>
+        )}
 
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter de Decker</p>
-            </div>
+        {/* <div className="dropdown-menu" role="menu">
+          <div className="dropdown-content">
```

Comment:
remove all comments

Reply:
<img width="632" alt="Screenshot 2023-12-02 at 12 30 08" src="https://github.com/mate-academy/react_autocomplete/assets/112933414/8c414512-b085-42ee-ad92-fadb1e460e22">


how do I hide a sheet if it is shown on focus?